pyrit/orchestrator/fuzzer_orchestrator.py

```diff
@@ -0,0 +1,393 @@
+                )
+
+            target_template = PromptTemplate(target_seed_obj, parameters=["prompt"])
+
+            # convert the target_template into a prompt_node to maintain the tree information
+            target_template_node = PromptNode(template=target_seed_obj, parent=None)
+
+            # 3. Fill in prompts into the newly generated template.
+            jailbreak_prompts = []
+            for prompt in self._prompts:
+                jailbreak_prompts.append(target_template.apply_custom_metaprompt_parameters(prompt=prompt))
+
+            # 4. Apply prompt converter if any and send request to the target
+            requests: list[NormalizerRequest] = []
+            for jailbreak_prompt in jailbreak_prompts:
+                requests.append(
+                    self._create_normalizer_request(
+                        prompt_text=jailbreak_prompt,
+                        prompt_type="text",
+                        converters=self._prompt_converters,
+                    )
+                )
+
+            for request in requests:
+                request.validate()
+
+            responses: list[PromptRequestResponse]
+            responses = await self._prompt_normalizer.send_prompt_batch_to_target_async(
+                requests=requests,
+                target=self._prompt_target,
+                labels=self._global_memory_labels,
+                orchestrator_identifier=self.get_identifier(),
+                batch_size=self._batch_size,
+            )
+
+            response_pieces = [response.request_pieces[0] for response in responses]
+            conversation_ids = [response.request_pieces[0].conversation_id for response in responses]
+
+            # 5. Score responses.
+            scores = await self._scorer.score_prompts_batch_async(
+                request_responses=response_pieces, tasks=self._prompts
+            )
+            score_values = [score.get_value() for score in scores]
+
+            # 6. Update the rewards for each of the nodes.
+            jailbreak_count = 0
+            for index, score in enumerate(score_values):
+                if score is True:
+                    jailbreak_count += 1
+                    self._jailbreak_conversation_ids.append(conversation_ids[index])
+            num_executed_queries = len(score_values)
+
+            self._total_jailbreak_count += jailbreak_count
+            self._total_target_query_count += num_executed_queries
+
+            if jailbreak_count > 0:
+                # The template resulted in at least one jailbreak so it will be part of the results
+                # and a potential starting template for future iterations.
+                self._new_templates.append(target_template.template)
+                target_template_node.add_parent(current_seed)
+
+            # update the rewards for the target node and others on its path
+            self._update(target_template_node, jailbreak_count=jailbreak_count)

```

Comment:
```suggestion
            self._update(jailbreak_count=jailbreak_count)

```

This arg is gone so it makes it fail. Please remove